projects/1.18/assets/tiny-redstone/tinyredstone/lang/zh_cn.json

```
@@ -1 +1,96 @@
-{}
\ No newline at end of file
+{
+  "key.tinyredstone.rotation_lock": "锁定旋转（键位+滚轮）",
+  "block.tinyredstone.redstone_panel": "红石板",
+  "block.tinyredstone.tiny_container": "微型元件容器",
+  "message.item.redstone_panel": "§7用于放置微型红石元件的面板。\n使用染料右击来改变颜色。§r",
+  "message.item.redstone_wrench": "§7右击红石板使其旋转。\n左击移除微型红石元件。\nShift+左击移除所有微型红石元件。§r",
+  "message.item.redstone_panel_cell": "可以被放置在红石板或者世界中。",
+  "message.item.tiny_color_selector": "§7右击微型方块来选择其颜色。\n兼容绝大多数方块型元件。§r",
+  "item.tinyredstone.tiny_redstone": "微型红石粉",
+  "message.item.tinyredstone.tiny_redstone": "传输红石信号，与原版红石粉类似。\n右击选择要连接的面。",
+  "item.tinyredstone.tiny_repeater": "微型中继器",
+  "message.item.tinyredstone.tiny_repeater": "后方接收到红石信号时，向前输出强度为15的强充能红石信号，可用右键调节延迟（1~4红石刻）。",
+  "item.tinyredstone.tiny_redstone_torch": "微型红石火把",
+  "message.item.tinyredstone.tiny_redstone_torch": "可以作为红石信号源和非门。\n像原版火把一样附着在微型方块上，但当位于面板或方块顶部时，可以向远离你的方向倾斜。\n因此无需将其放在方块上便可制作非门。",
+  "item.tinyredstone.redstone_wrench": "微型元件板手",
```

```suggestion
  "item.tinyredstone.redstone_wrench": "微型元件扳手",
```
:rofl:
L53还有一处，建议全文替换（除非我想错了）

---

草，为什么会出现这种问题😩